php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -500,7 +500,7 @@ class NDB_BVL_Instrument extends NDB_Page
 
                 // the form WAS submitted but validate() failed...
                 // main error message
-                $this->form->addElement('static', 'mainError');
+                $this->form->addElement('static', 'mainError', '');
```

Comment:
@davidblader wouldn't it be better to modify the addElement() function definition to default to `''` when no argument is given ??

so you would have something like 
```
public function addElement(
        $type,
        $name,
        $label='',
        $options=array(),
        $attribs=array()
    ) {
```